content/portfolio.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<!-- head -->
+  <head>
+    <meta charset="utf-8">
+    <title>About</title>
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:300"
+      rel="stylesheet">
+    <link rel="stylesheet" href="../styles/normalize.css">
+    <link rel="stylesheet" href="../styles/style.css">
+  </head>
+
+<!-- body -->
+  <body>
+
+<!-- entire-site-container -->
+    <div class="entire-site-container">
+
+<!-- nav -->
+      <nav class="nav">
+        <ul>
+          <li><a href="../index.html">home</a></li>
+          <li><a href="about.html">about</a></li>
+          <li><a href="portfolio.html">portfolio</a></li>
+          <li><a href="hobby-blog.html">blog</a></li>
+        </ul>
+      </nav>
+
+<!-- picture area -->
+      <div class="picture-main">
+        <ul>
+          <li><h1>Kirsten Schumy</h1></li>
+          <li>Hater of the Chicago Bears and FlexBox</li>
+        </ul>
+      </div>
+
+<!-- page title -->
+      <div class="page-title">
+        <h2>Portfolio</h2>
+      </div>
+
+<!-- content area -->
+      <div class="content-main">
+        <div>
+          <ul class="portfolio-layout">
+            <li>
+              <p>
+                Ruby at Ada
+                <img src="../images/ruby2.png" alt="example of ruby">
+              </p>
+            </li>
+            <li>
+              <p>
+                Ruby at Ada
+                <img src="../images/ruby2.png" alt="example of ruby">
+              </p>
+            </li>
+            <li>
+              <p>
+                HTML/CSS
+                <img src="../images/starterly-shot.png" alt="example of html">
+              </p>
+            </li>
+            <li>
+              <p>
+                CSE 142, Computer Programming I
+                <img src="../images/cse142-copy.png" alt="example of java">
+              </p>
+            </li>
+            <li>
+              <p>
+                CSE 143, Computer Programming II
+                <img src="../images/cse143-copy.png" alt="example of java">
+              </p>
+            </li>
+            <li>
+              <p>
+                CSE 373, Data Structures & Algorithms
```

Comment:
The validator might have been thrown off by the `&` character.  In HTML proper you would put `&amp;`.  

It could also be that you have two identical `alt` attributes and it's warning you of those.